doc/SWD.md

```diff
@@ -4,9 +4,9 @@ Download the files **bootloader.bin**, **image-x.y.z.bin** and **pinetime-graphi
 ![Image file](imageFile.png)
 
 The bootloader reads a boot logo from the external SPI flash memory. The first step consists of flashing a tool in the MCU that will flash the boot logo into this SPI flash memory. This first step is optional but recommended (the bootloader will display garbage on screen for a few second if you don't do it).
-Using your SWD tool, flash **pinetime-graphics-x.y.z.bin** at offset **0x0000**. Reset the MCU and wait for a few second, until the logo is completely drawn on the display.
+Using your SWD tool, flash **pinetime-graphics-x.y.z.bin** at offset **0x0000**. Reset the MCU and wait for a few seconds until the logo is completely drawn on the display.
 
-Then, using your SWD tool, flash those file at specific offset:
+Then, using your SWD tool. Flash those file at specific offset:
```

Comment:
I think this was more correct before the change. Also the wording could be improved like for example "flash the following files at their specific offsets".